blocks/library/cover-image/index.js

```diff
@@ -152,22 +177,69 @@ registerBlockType( 'core/cover-image', {
 				className={ classes }
 			>
 				{ title || !! focus ? (
-					<Editable
-						tagName="h2"
-						placeholder={ __( 'Write title…' ) }
-						value={ title }
-						focus={ focus }
-						onFocus={ setFocus }
-						onChange={ ( value ) => setAttributes( { title: value } ) }
-						inlineToolbar
+					<div className="title-container">
+						<Editable
+							tagName="h2"
+							placeholder={ __( 'Write title…' ) }
+							value={ title }
+							focus={ focusedEditable === 'title' ? focus : null }
+							onFocus={ ( props ) => setFocus( { ...props, editable: 'title' } ) }
+							onChange={ ( value ) => setAttributes( { title: value } ) }
+							inlineToolbar
 					/>
+					</div>
 				) : null }
+
+				{ hasButton &&
+					<span key="button" className="button-container aligncenter" style={ { backgroundColor: buttonBackgroundColor } } >
+						<Editable
+							tagName="span"
+							placeholder={ __( 'Add text…' ) }
+							value={ buttonText }
+							focus={ focusedEditable === 'button' ? focus : null }
+							onFocus={ ( props ) => setFocus( { ...props, editable: 'button' } ) }
+							onChange={ ( value ) => setAttributes( { buttonText: value } ) }
+							formattingControls={ [ 'bold', 'italic', 'strikethrough' ] }
+							style={ {
+								color: buttonTextColor,
+							} }
+							keepPlaceholderOnFocus
+						/>
+						{ focusedEditable === 'button' &&
+							<form
+								className="blocks-format-toolbar__link-modal"
+								onSubmit={ ( event ) => event.preventDefault() }>
```

Comment:
Why do we render this as a form if we're not doing anything form-like with it? Seems we could just as easily drop the form and avoid assigning `type="submit"` to the icon button.